services-js/registry-certs/client/store/BirthCertificateRequest.ts

```diff
@@ -53,6 +71,29 @@ export default class BirthCertificateRequest {
     this.currentStepCompleted = isCompleted;
   };
 
+  @action
+  public addVerificationStep = (): void => {
+    const targetIndex = this.steps.indexOf('parentalInformation') + 1;
+
+    if (!this.steps.includes('verifyIdentification')) {
+      this.steps.splice(targetIndex, 0, 'verifyIdentification');
```

Comment:
Warning here that you’re mutating `this.steps`, which was initialized with a global constant. This means that these changes are affecting any future `BirthCertificateRequest` objects that are created.

It might not be a bug here in practice, but it is one of those things that could surprise you later.

Reply:
I started off with `this.steps = this.steps.splice(etc)` but found that when the `verify` step was added to the array, the progress bar component lost the information... I should have left a comment! 😞